### DISCUSSION TITLE
New Major Release with macOS 12.3+ support

### DISCUSSION THREAD

**Discussion by skleinei** [skleinei/alfred-confluence#16] [Announcements]
New major version that now works with macOS 12.3 (Monterey) and up.

**Notable Improvements:**
* Filter by spaces, example: `c search term -s SPACEKEY` (default: search all spaces)
* Filter by content types, example: `c search term -t attachment` (default: page,blogpost)
* Increase results limit, example: `c search term -l 50` (default: 10)
* Use ⌘ as modifier key to directly open page in editor

**🚨 Breaking Changes:**
* Removed support for Confluence Server/DataCenter (we may add this later)
* Bumped requirement to Alfred 5, because it is using the new workflow configuration UI.

<hr /><em>This discussion was created from the release <a href='https://github.com/skleinei/alfred-confluence/releases/tag/2.0.0'>New Major Release with macOS 12.3+ support</a>.</em>

**Comment by dmittman** [comment]
How "difficult" (recognizing that that's hard to quantify) would it be to restore compatibility with Confluence Server/Data Center? Unifying intranet search (including Confluence Server/Data Center) is a critical use case for me for Alfred. If you removed support for Confluence Server/Data Center as part of an architectural overhaul, I imagine adding it back won't be easy. If it was just a matter of access to Confluence Server/Data Center test instances, then it might be "easier." In any case, I'm happy to contribute to restoring Confluence Server/Data Center compatibility if it's on the "easy" end of the spectrum.

**Reply by skleinei** [reply]
@dmittman thanks for your feedback. It's definitely doable. I've created an [issue](https://github.com/skleinei/alfred-confluence/issues/17) to track interest in this.

**Reply by skleinei** [reply]
@dmittman and everyone else: https://github.com/skleinei/alfred-confluence/releases/tag/2.0.1 with Data Center and Service support is out.
